### ISSUE TITLE
fix filename parser for windows

### ISSUE THREAD

**Issue by dcsan** [dcsan/gitterbot#93]
in KBase.js

```
                var arr = filePath.split("/");  // FIXME for windows
                var fileName = arr[arr.length - 1];
```

links currently look like this:

```
 c:\projects\gitterbot\nap\data\wiki\js indexof
 c:\projects\gitterbot\nap\data\wiki\js loops
```

**Comment by oab00** [comment]
this was fixed in #100  (special number) ;)

**Comment by dcsan** [comment]
thanks for the housekeeping!
